addons/rating/CfgEventHandlers.hpp

```diff
@@ -0,0 +1,25 @@
+class Extended_Init_EventHandlers {
+    class CAManBase {
+        class ADDON {
+            serverInit = QUOTE(_this call FUNC(initMan));
```

Comment:
`initMan` checks if the settings init is finished and adds all pending requests to the settings finished execution queue. Don't really see a benefit adding this on the script side.